app/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,29 @@
+import { cn } from "../../lib/utils/cn";
+
+type DialogProperties = {
+  headerText: string;
+  description?: string;
+  footer: React.ReactNode[];
+};
+const Dialog = ({ headerText, description, footer }: DialogProperties) => {
+  return (
+    <div className="flex flex-col p-6 gap-4 bg-white rounded-[6px] max-w-[512px] w-auto">
+      <header className="flex flex-col gap-2">
+        <h2 className="text-lg font-semibold text-primary">{headerText}</h2>
+        {description && (
+          <p className="font-normal text-sm leading-[1.05rem] text-muted-foreground text-pretty">
```

Comment:
Why are you creating your own dialog? Weren't you told to use ShadCN dialog component?

Reply:
In the issue requirements, I wasn't asked to use Shadcn-UI.

The Shadcn-UI Dialog component requires a close icon to close the dialog and a trigger button to display it, but these were not mentioned in the requirements and also not in the figma file.

I have attached the figma link for the component in the PR description for your reference.

Should I still go ahead and implement them anyway?

Reply:
Meet @SirhmVFX on slack to install it. he's @fullstackmechanic on slack